pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_identifying-module-types.adoc

```diff
@@ -0,0 +1,46 @@
+[id='identifying-module-types_{context}']
+= Identifying {ProductShortName} module types
+:pantheon-module-type: REFERENCE
+
+In Pantheon 2, each module must be identified by type: concept, procedure, or reference.
+
+Module types are described in https://redhat-documentation.github.io/modular-docs/#writing-mod-docs[Types of modules in Red Hat documentation].
+
+Identification is done by using either one or both of the following:
+
+* The correct attribute after the module's title
+* The correct prefix in the module's name
+
+Use of the correct attribute (preferred method)::
```

Comment:
Using an attribute

Reply:
That sounds like a procedure heading to me. This is a reference module.

Reply:
Not necessarily. The guidance I've seen is that when using a verb, use the gerund form. If you want to structure a heading title (btw, this is a not really a heading, it's a definition/description term) to not use the gerund, then you shouldn't use a verb at all.